client/src/index.js

```diff
@@ -1,39 +1,35 @@
-import React from "react";
+import React, { useState } from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
 import Home from "./components/Home";
 import "semantic-ui-css/semantic.min.css";
 import About from "./components/About";
 import Status from "./components/Status";
-import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import { NavLink, BrowserRouter as Router, Route } from "react-router-dom";
+import { Menu, Container, Image, Input } from "semantic-ui-react";
 
 const Routes = () => {
   return (
     <Router>
-      <div>
-        <ul>
-          <li>
-            <Link className="nav-link" to="/">
-              Home
-            </Link>
-          </li>
-          <li>
-            <Link className="nav-link" to="/about">
-              About
-            </Link>
-          </li>
-          <li>
-            <Link className="nav-link" to="/status">
-              Status
-            </Link>
-          </li>
-        </ul>
+      <Menu inverted>
+        <Container>
+          <Menu.Item header>
+            <Image
+              size="small"
+              src="http://www.maryhillintegration.org.uk/wp-content/uploads/2016/10/banner.jpg"
```

Comment:
It's generally not a good idea to hardcode an image URL in code because we don't have control over the image. What if the image on maryhillintegration.org.uk is removed? Then our project shows a broken image.

What is normally done instead is that the image is stored in the project. We can save `banner.jpg` in our project, for example at path `client/src/assets/banner.jpg` then import it to use here.

Also, web accessibility is something that is becoming more discussed nowadays. One thing that is strongly suggested is that all images should have an `alt` attribute which describes what the image is, for example `alt="Maryhill Integration logo"`

Reply:
Thank you so much for your helpful comments. firstly I put banner.jpg in the projct but when pushing the code i got this error "(Luke was helping me so he knows about this issue)
```
✖ prettier-eslint --write found some errors. Please fix them and try committing again.
prettier-eslint [ERROR]: prettier formatting failed due to a prettier error
prettier-eslint-cli [ERROR]: There was an error formatting "/home/coder/Desktop/cyf/js/scot-min-imece/client/src/styles/banner.jpg": 
  Error: No parser could be inferred for file: /home/coder/Desktop/cyf/js/scot-min-imece/client/src/styles/banner.jpg".
```
That is why i had to use the URL.

Reply:
I think Rares fixed this with a new commit in the `master` branch. Can you update to the latest `master` and try again? I think it should work then

Reply:
@hacersk I think Rares fixed this with a new commit in the `master` branch. Can you update to the latest `master` and try again? I think it should work then

Reply:
@hacersk Yes, I think this is fixed on master. To bring your branch up to date with the latest changes from master (make sure you don't have any uncommitted changes first):

```
git checkout master
git pull
git checkout hacer/update-menu
git merge master
```
You can run `git help merge` to see more on what that last command does.

I'd suggest you create a new folder `client/src/assets` and add a copy of the logo file in there. Then push the changes.